Microsoft.Toolkit/Observables/ObservableGroup.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Microsoft.Toolkit.Observables.Collections
+{
+    /// <summary>
+    /// An observable group.
+    /// It associates a <see cref="Key"/> to an <see cref="ObservableCollection{T}"/>.
+    /// </summary>
+    /// <typeparam name="TKey">The type of the group key.</typeparam>
+    /// <typeparam name="TValue">The type of the items in the collection.</typeparam>
+    public sealed class ObservableGroup<TKey, TValue> : ObservableCollection<TValue>, IGrouping<TKey, TValue>, IReadOnlyObservableGroup
```

Comment:
We're also missing things like `IEnumerable` and `ICollectionViewGroup`?

Reply:
`IEnumerable<T>` (and `IEnumerable`) are already implemented by `ObservableCollection<TValue>`, so there's no need to declare that here as well (this doesn't apply to `ICollectionViewGroup`) 😊

Reply:
I'm not implementing `ICollectionViewGroup` because the code is in the *.Net standard* project and the interface is not available. 
I think it will be better to keep it this way to allow any .Net project to consume the collections and not only the UWP ones. This is useful when sharing code (like view models) between UWP and a Xamarin app.